homeassistant/components/persistent_notification/__init__.py

```diff
@@ -97,6 +117,12 @@ def async_dismiss(hass: HomeAssistant, notification_id: str) -> None:
 
 async def async_setup(hass: HomeAssistant, config: dict) -> bool:
     """Set up the persistent notification component."""
+    # Process configs
+    if DOMAIN in config:
```

Comment:
I don't think that we should put anything inside `persistent_notification`. I think that instead we should just add this notification service to `notify/__init__.py` and just always set up as `notify.persistent_notification`.